.github/workflows/release-binaries.yml

```diff
@@ -156,17 +156,29 @@ jobs:
         sparse-checkout: |
           .github/workflows/
         sparse-checkout-cone-mode: false
+        # Check out outside of working directory so the source checkout doesn't
+        # remove it.
         path: workflows
 
-    - name: Setup Stage
-      id: setup-stage
-      uses: ./workflows/.github/workflows/release-binaries-setup-stage
+    # Move workflows so they don't get overwritten by checkout.  The checkout job
+    # requires checking out to github.workspace.
+    - shell: bash
```

Comment:
Why do we need to move it again if we're already checking out to a different path?

Reply:
It wouldn't let me specify a `uses` path that was outside of the github workspace.

Reply:
Can you add/modify the comment to better clarify this requirement?

Reply:
OK, I've added a better description.

Reply:
Thank you!